package.json

```diff
@@ -16,6 +16,9 @@
     "build": "rm -rf dist && webpack --config webpack.config.js --mode production",
     "packdemo": "cd ./example && npm run pack",
     "watch": "webpack --config webpack.config.js --mode development --watch",
+    "watch:demo": "cd ./example && npm run watch",
+    "watch:web": "run-p watch watch:demo watch:serve",
+    "watch:serve": "live-server --port=9000 example/dist",
```

Comment:
nit: some naming considerations:
I'd prefer "watch:demo" renamed to "watch:example" for terminology consistency in the repo. And "watch:serve" renamed to "serve:example".

Reply:
This can be done.
Will push a commit this morning for review